### DISCUSSION TITLE
help for mutliple items

### DISCUSSION THREAD

**Discussion by kkcroft71** [arkypita/LaserGRBL#1845] [Q&A]
I am a beginner and i have a question i hope someone can help me with. Can i cut multiple small items at the same time instead of one by one. For example i want to burn some wood magnets, they will all be the same. So can i burn several at one time or would i have to do them one by one?

**Comment by Tribo-2dot0** [comment]
Mount the svg in Inkscape. Make one, copy, paste and arrange the way you see fit. Save the svg and open it in LaserGrbl.

**Comment by kkcroft71** [comment]
I don't have Inkscape and don't understand, like I said I am a beginner.

**Comment by PhilliasFogg** [comment]
Just Load Your File Into LaserGrbl then under file menu hit Append File & Load it again and use the offset to place it next to your previous image, you can keep doing this until you have as many as you need then save the file for future use
